scripts/config/notebook-testing.toml

```diff
@@ -28,7 +28,7 @@ notebooks_normal_test = [
     "docs/guides/simulate-with-qiskit-aer.ipynb",
     "docs/guides/simulate-stabilizer-circuits.ipynb",
     "docs/guides/operator-class.ipynb",
-    "docs/guides/error-mitigation-and-suppression-techniques.ipynb",        
+    "docs/guides/error-mitigation-and-suppression-techniques.ipynb",
     "docs/guides/error-mitigation-and-suppression-techniques.ipynb",
```

Comment:
is the same notebook supposed to be repeated twice here?

Reply:
Ooh - good catch!  Thanks!